### PR TITLE
Upgrade bwa_mem_index_builder's bwa dependency to 0.7.12

### PR DESCRIPTION
**Pull request by mvdbeek**
Since 0.7.10.039ea20639 is not availble for OS X.

### CODE REVIEW

File: data_managers/data_manager_bwa_mem_index_builder/data_manager/bwa_mem_index_builder.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <tool id="bwa_mem_index_builder_data_manager" name="BWA-MEM index" tool_type="manage_data" version="0.0.1">
```
Comment (nsoranzo) [inline data_managers/data_manager_bwa_mem_index_builder/data_manager/bwa_mem_index_builder.xml]: Increase tool version.

**Comment by bgruening** [comment]
Thanks Marius!

**Comment by mvdbeek** [comment]
Sure! @martenson can you upload this to the toolshed?

**Comment by martenson** [comment]
done, thanks @mvdbeek @bgruening

**Pull request merged**